valhalla/sif/edgelabel.h

```diff
@@ -444,12 +459,17 @@ class EdgeLabel {
   // Flag indicating edge is an unpaved road.
   uint32_t unpaved_ : 1;
   uint32_t has_measured_speed_ : 1;
+
   // Flag if this edge had HGV access
   uint32_t hgv_access_ : 1;
   uint32_t spare : 12;
 
   Cost cost_;      // Cost and elapsed time along the path.
   float sortcost_; // Sort cost - includes A* heuristic.
+                   // Flag indicating edge is a bridge.
```

Comment:
probably better for legibility to move this just above the spare

Reply:
do you mean // Flag indicating edge is a bridge.
  uint32_t bridge_ : 1;
  // Flag indicating edge is a tunnel.
  uint32_t tunnel_ : 1; after hgv_access and before spare?

Reply:
Done